packages/desktop/src/lib.rs

```diff
@@ -187,6 +187,25 @@ pub fn launch_with_props<P: 'static + Send>(
                                 is_ready.store(true, std::sync::atomic::Ordering::Relaxed);
                                 let _ = proxy.send_event(UserWindowEvent::Update);
                             }
+                            "browser_open" => {
+                                let data = req.params.unwrap();
+                                log::trace!("Open browser: {:?}", data);
+                                if data.is_array() {
+                                    let arr = data.as_array().unwrap();
+                                    if arr[0].is_object() {
+                                        let temp = arr[0].as_object().unwrap();
+                                        if temp.contains_key("href") {
```

Comment:
I'm concerned about all the unwrapping and bare indexing. Can you refactor this to use `.map` or `if let Some(_) = `?

There's still a few unwraps laying around that cause issues in other parts of Dioxus that should be `if let Some(_)`